src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -7,6 +7,7 @@ Release History
 ++++++
 * `image create`: expose storage-sku argument for setting the image's default storage account type
 * `vm resize`: fix bug where `--no-wait` option causes command to crash
+* `vm encryption show`: table output format shows status for encryption initiated using `--disk-encryption-keyvault`
```

Comment:
I suggest dropping the `initiated using --disk-encryption-keyvault` because it is irrelevant.